tapiriik/local_settings.py.example

```diff
@@ -7,6 +7,9 @@ DROPBOX_FULL_APP_SECRET = "####"
 DROPBOX_APP_KEY = "####"
 DROPBOX_APP_SECRET = "####"
 
+GOOGLEDRIVE_CLIENT_ID = "####"
```

Comment:
By the way, you can create projects and configure the APIs they use at https://console.developers.google.com